migrate/20231012_dns_zones.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:dns_zone) do
+      column :id, :uuid, primary_key: true, default: nil
+      column :created_at, :timestamptz, null: false, default: Sequel.lit("now()")
+      column :project_id, :project, type: :uuid, null: false
+      column :location, :text, collate: '"C"', null: false
+      column :name, :text, collate: '"C"', null: false
```

Comment:
Thinking on this more... I didn't want to move name column to the join table, because name information is not part of the join relationship. I think it is better to do it in the application side.

Reply:
Thanks for such a detailed explanation 🚀